ReactiveCocoaTests/Swift/SignalSpec.swift

```diff
@@ -453,6 +473,33 @@ class SignalSpec: QuickSpec {
 				observer.sendNext(1)
 				expect(lastValue).to(equal("2"))
 			}
+
+			it("should not keep resulting signal alive indefinitely") {
+				weak var signal: Signal<AnyObject, NoError>? = Signal.never.map { $0 }
+
+				expect(signal).to(beNil())
+			}
+
+			it("should not keep resulting signal alive indefinitely even if it has an observer") {
```

Comment:
Sorry but this changes [the semantics of `Signal`](https://github.com/ReactiveCocoa/ReactiveCocoa/blob/23dab6c3e69a64fb4d027e29ff90ac6896a7bac8/ReactiveCocoa/Swift/Signal.swift#L15-L16).`